<launch>
	<arg name="node_name" default="text_recognizer_node"/>
	<group ns="/ubiquityrobot">
		<remap from="$(arg node_name)/image_raw/compressed" to="camera_node/image_raw/compressed" />
		<remap from="$(arg node_name)/image_raw" to="camera_node/image_raw" />
		<remap from="$(arg node_name)/camera_get_frame" to="camera_node/camera_get_frame" />
		<node name="$(arg node_name)" pkg="pi_cam" type="$(arg node_name).py" output="screen" >
		</node>
	</group>
</launch>